auth-web/tests/unit/setup.ts

```diff
@@ -22,3 +22,12 @@ Vue.directive('can', can)
 document.body.setAttribute('data-app', 'true')
 
 setActivePinia(createPinia())
+
+// Mock BaseAddress component to avoid decorator issues in tests
+vi.mock('@bcrs-shared-components/base-address/BaseAddress.vue', () => ({
```

Comment:
this is solving:
<img width="893" alt="image" src="https://github.com/user-attachments/assets/109fea4a-ff55-45e9-bd9f-eedf0a51ec8f" />